### PR TITLE
Render latex on php mode

### PR DESCRIPTION
**Pull request by xjiang-at-wiris**
## Description

> Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change. Include, if necessary, a description of the proposed solution and the reasoning behind it.

The ability of MathType to render LaTeX was removed at some point, and now it needs to be added back.

## Type of Change

> Please delete options that are not relevant.

- [ ] Feature (non-breaking change which adds functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (non-breaking change that doesn't add any functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (changes to documentation only)
- [ ] Refactoring (non-breaking change that improves the code structure)

## How should be tested? (Manual or Automated Tests)
 
- Launch moodle
- Enure your render type is PHP ([link](http://localhost:8000/admin/search.php?query=render+type))
- Write MathML and LateX son some page
- Play with `wiriseditorparselatex` parameter (true or false)
   - On file `configuration.ini` 
- Play with Mathjax priority [Link](http://localhost:8000/admin/filters.php)
Compare the results; they should follow the result shown in the table.

| ID | Render | wirisparselatex                 | MathJax        | Latex Renderer Expected |
|----|--------|----------------------------------|----------------|-------------------------|
| 1  | PHP    | TRUE                             | Priority over MathType | MathJax                |
| 2  | PHP    | TRUE                             | MathType has priority  | MT                     |
| 3  | PHP    | TRUE                             | Disabled       | MT                      |
| 4  | PHP    | FALSE                            | Priority over MathType | MathJax                |
| 5  | PHP    | FALSE                            | MathType has priority  | MathJax                |
| 6  | PHP    | FALSE                            | Disabled       | No render              |

[Task  #50947](https://wiris.kanbanize.com/ctrl_board/2/cards/50947/details/)

### CODE REVIEW

File: subfilters/php.php [hunk 1]
```diff
@@ -147,7 +161,6 @@ private function mathjax_have_preference() {
                 $mathjaxpreference = true;
             }
         }
-
```
Comment (jgonzalez-at-wiris) [inline subfilters/php.php:150]: ¿Por qué has quitado el salto de línea? 😢 😂

**Review by jgonzalez-at-wiris — APPROVED** [review]
No lo he probado, pero a nivel de código me parece que está genial! Gran trabajo Xinyu!

File: subfilters/php.php (hunk 1 above)
Comment (xjiang-at-wiris) [reply subfilters/php.php:150]: Puse un echo para ver el resultado jajaja, voy a ponerlo de vuelta

File: subfilters/php.php [hunk 2]
```diff
@@ -56,7 +56,12 @@ public function filter($text, array $options = []) {
         $n1 = stripos($text, '<math');
         $n2 = mb_stripos($text, '«applet');
 
-        if ($n0 === false && $n1 === false && $n2 === false) {
+        // find LateX
```
Comment (usantos-at-wiris) [inline subfilters/php.php:59]:
```suggestion
        // Find LateX.
```

File: subfilters/php.php [hunk 3]
```diff
@@ -89,6 +88,20 @@ public function filter($text, array $options = []) {
             $query .= 'category=' . $COURSE->category;
         }
 
+        // If MathJax don't have preference and wiriseditorparselatex = true, parse LateX into MathML
```
Comment (usantos-at-wiris) [inline subfilters/php.php:91]:
```suggestion
        // If MathJax don't have preference and wiriseditorparselatex = true, parse LateX into MathML.
```

**Review by usantos-at-wiris — CHANGES_REQUESTED** [review]

File: subfilters/php.php (hunk 3 above)
Comment (jgonzalez-at-wiris) [inline subfilters/php.php:91]:
```suggestion
        // If MathJax doesn't have preference and wiriseditorparselatex = true, parse LateX into MathML
```

**Review by carla-at-wiris — APPROVED** [review]

**Pull request merged**